.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.env
```

Comment:
Why didn't `.env` get ignored?

Reply:
Thanks for the catch, I likely committed at one point and pushed, then later added the .env to gitignore, I deleted the  api keys.